e2e/cloudwalk-contracts/contracts-clone.sh

```diff
@@ -173,6 +177,8 @@ if [ "$pix" == 1 ]; then
 fi
 
 if [ "$yield" == 1 ]; then
+    # BalanceTracker Transition: clone balance tracker if it exists and contains the hardhat project.
+    clone brlc-balance-tracker || log "Balance Tracker not isolated yet. Skipping..."
```

Comment:
The following part is redundant:
```
|| log "Balance Tracker not isolated yet. Skipping..."
```
because the repo already exists, so it will be cloned anyway, but tests and other commands may fail till the migration is ready

Reply:
Considering that the `clone` function here also runs `npm install`, this part is important to not fail the entire script if the repository exists but does not contain a hardhat/node project. I believe it could be better, but after all the renamings, I will just remove those "transition code"

Reply:
Sorry, I missed the `npm install` part.
Now I agree :)